.github/workflows/testing.yml

```diff
@@ -63,3 +63,47 @@ jobs:
           COVERALLS_TOKEN: ${{ secrets.GITHUB_TOKEN }}
         run: |
           make coveralls
+  run-tests-ee:
+    if: github.event_name == 'push' ||
+      github.event.pull_request.head.repo.full_name != github.repository
+
+    runs-on: ubuntu-latest
+
+    env:
+      TEST_TNT_EE: 1
+
+    strategy:
+      fail-fast: false
+      matrix:
+        sdk-version: ["2.10.0-beta2-59-gc51e2ba67-r469"]
+
+    steps:
+      - name: Clone the connector
+        uses: actions/checkout@v2
+
+      - name: Setup Tarantool ${{ matrix.sdk-version }}
+        run: |
+          ARCHIVE_NAME=tarantool-enterprise-bundle-${{ matrix.sdk-version }}.tar.gz
+          curl -O -L https://${{ secrets.SDK_DOWNLOAD_TOKEN }}@download.tarantool.io/enterprise/${ARCHIVE_NAME}
+          tar -xzf ${ARCHIVE_NAME}
+          rm -f ${ARCHIVE_NAME}
+          sudo cp tarantool-enterprise/tarantool /usr/bin/tarantool
```

Comment:
I think it's better to use `source tarantool-enterprise/env.sh` where needed

Reply:
Thank you, replaced.